apps/charterafrica/src/lib/data/fetchApi.js

```diff
@@ -0,0 +1,68 @@
+import qs from "qs";
```

Comment:
Do we need any of this **_now_**?

Reply:
We need a util function that converts nested objects searchParams.

`qs` does that.

Or we have to write a util function to handle that.

Reply:
@kilemensi

Reply:
Not sure I follow @koechkevin: How do nested search params look like in a URL?

Reply:
We need to send a query with params object that looks like this 
```js
{
  locale: 'en',
  'fallback-locale': 'en',
  locales: [ 'en', 'fr', 'pt' ],
  where: { slug: { equals: '404' } }
}
```
We then expect a url like `?locale=en&fallback-locale=en&locales%5B0%5D=en&locales%5B1%5D=fr&locales%5B2%5D=pt&where%5Bslug%5D%5Bequals%5D=404`

but URLSearchParams ends up with something like `'?locale=en&fallback-locale=en&locales=en%2Cfr%2Cpt&where=%5Bobject+Object%5D'`

Reply:
Damn, that's more like GraphQL-ing search params @koechkevin! `...&slug=404` should have been enough for REST API.

... but if that's the structure Payload expects, there is nothing we can do now.

🚀